src/rust/src/rdatatype.rs

```diff
@@ -73,7 +73,7 @@ impl RPolarsDataType {
             "Float32" | "float32" | "double" => pl::DataType::Float32,
             "Float64" | "float64" => pl::DataType::Float64,
 
-            "String" | "character" => pl::DataType::String,
+            "Utf8" | "String" | "character" => pl::DataType::String,
```

Comment:
I was just getting started on this too. I think we can leave Utf8 as an alias, as py-polars does the same thing.

Reply:
Yes, we should keep an alias but I guess docs should be updated to prefer "String" instead

Reply:
Go ahead if you want, let me know when I can jump in ;)